{"@class": "de.fau.cs.inf2.cthree.data.Cluster", "id": "f4805dd5dc1b0b4970621ed976135a28", "detectedBy": ["AST_DBSCAN"], "members": [{"@class": "de.fau.cs.inf2.cthree.data.CodeChange", "id": "2e27586a0e5539388f1ee224337777d0", "repository": "https://github.com/checkstyle/checkstyle.git", "fileName": "src/tests/com/puppycrawl/tools/checkstyle/CheckerTest.java", "commitBeforeChange": "5703b6e33550d6e9706f42e253c2cc7d83a358bf", "commitAfterChange": "1818ae8ab189515f24a711aa3fc520377bd71380", "methodNumberBeforeChange": 22, "methodNumberAfterChange": 22, "signatureBeforeChange": "     public void testRelaxedJavadoc()         throws Exception", "signatureAfterChange": "     public void testRelaxedJavadoc()         throws Exception", "diff": ["-            filepath + \":45:19: Variable 'mDeer' missing Javadoc.\",", "+            filepath + \":45:19: Missing a Javadoc comment.\",", "-            filepath + \":46:16: Variable 'aFreddo' missing Javadoc.\",", "+            filepath + \":46:16: Missing a Javadoc comment.\","]}, {"@class": "de.fau.cs.inf2.cthree.data.CodeChange", "id": "e41b8138a55b00103d60fd19bc1d2754", "repository": "https://github.com/checkstyle/checkstyle.git", "fileName": "src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalCatchCheckTest.java", "commitBeforeChange": "ab50d42d93880d48c8aef9c88c37174bd941463f", "commitAfterChange": "448a62c9a6740c4e6e5435b4a89cd9c82e6b8957", "methodNumberBeforeChange": 2, "methodNumberAfterChange": 2, "signatureBeforeChange": "      @Test     public void testMultipleTypes() throws Exception", "signatureAfterChange": "      @Test     public void testMultipleTypes() throws Exception", "diff": ["-            \"6:11: Catching 'RuntimeException' is not allowed.\",", "-            \"8:11: Catching 'RuntimeException' is not allowed.\",", "+            \"7:11: Catching 'RuntimeException' is not allowed.\",", "-            \"12:11: Catching 'RuntimeException' is not allowed.\",", "+            \"13:11: Catching 'RuntimeException' is not allowed.\",", "+            \"16:11: Catching 'RuntimeException' is not allowed.\","]}]}